README.md

```diff
@@ -4,3 +4,33 @@ This repository holds the Go code used by the `hashicorp/consul-ecs` Docker imag
 This image is used to help with the installation and operation of Consul on ECS.
 
 See [https://consul.io/docs/ecs](https://consul.io/docs/ecs) for more information.
+
+## Consul on Roadmap
+
+Knowing about our upcoming features and priorities helps our users plan. This repository contains information about what we are working on and allows all our user to give direct feedback.
+
+
+[See the roadmap »](https://github.com/hashicorp/consul-ecs/projects/1)
+
+## Roadmap FAQs
+**Why did you build this public facing roadmap?**
+
+We know that our customers are making decisions and plans based on what we are developing, and we want to provide our customers the insights they need to plan.
```

Comment:
Not just customers, users too. I'd say users.

Reply:
Updated to reflect customers and users